### PR TITLE
Re-maps bigderelict1 ruin, culling its loot

### PR DESCRIPTION
**Pull request by Capsandi**
## About The Pull Request
Old:
image

New:
image

Changes of note;
1.  Var-edit mess that was the green creature cocoon has been replaced with spiders(most are scrawny)
2.  Entire ruin has been remapped, shrunk and it's layout has been simplified
3. The dead officer's outfit has been replaced with one without a security officer ID
4. The dozens of free crates of medical supplies/materials have been replaced with more tame loot and lootdrop spawners
5.  The M1911 pistol which the ruin is most known for now starts out with an outback firing pin, which is unable to be used on station. This can be replaced with a 200 credit-per-shot paywall pin which lies in a crate nearby
6. There is now a dock for the caravan cargo ship. This replaces the whiteship ruin dock in its port list

## Why It's Good For The Game

Security access from space is a problem, and the ruin had quite a bit of "soul" in its layout
I also felt that the gun was far too easy to grab without any drawbacks, and now would be a good time to swap it out with a laser gun if that is requested

## Changelog
:cl:
add: The abandoned tradepost ruin has been remapped and polished
balance: The loot in the abandoned trade-post ruin has been culled
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
Isn't this more of a wholesale replacement of the ruin rather than a remap?

**Comment by Capsandi** [comment]
> Isn't this more of a wholesale replacement of the ruin rather than a remap?

Well it looked like shit and had a ton of stuff you shouldn't be able to find in space. So I tried to keep the gist of it intact. If this is more of a replacement then I suppose a replacement was warranted

File: code/game/area/areas/ruins/space.dm [hunk 1]
```diff
@@ -116,10 +116,6 @@
 
 //Ruin of Derelict Oupost
 
-/area/ruin/space/has_grav/derelictoutpost
-	name = "Derelict Outpost"
-	icon_state = "green"
-
```
Comment (Fikou) [inline code/game/area/areas/ruins/space.dm:122]: why did you remove the parent type

File: code/modules/ruins/spaceruin_code/bigderelict1.dm [hunk 2]
```diff
@@ -6,3 +6,13 @@
 /obj/item/paper/crumpled/ruins/bigderelict1/coward
 	icon_state = "scrap_bloodied"
 	info = "If anyone finds this, please, don't let my kids know I died a coward.."
+
+/datum/outfit/tradepostcoward
+	name = "derelict tradepost officer"
```
Comment (Fikou) [inline code/modules/ruins/spaceruin_code/bigderelict1.dm:11]: this is like the only non capitalized outfit name in the game

**Comment by Fikou** [comment]
its way more boring imo. instead of creatures you cant find anywhere else you fight spiders that you can already find in like 2 other ruins, theres no more abandoned crates, removing a puzzle that gave some nice random loot if you were good, the cool idea of a broken shuttle that docked there got removed
using new mapping practices for ruins is nice, but sacrificing how interesting the ruin is for that is lame

**Comment by itseasytosee** [comment]
"qol"

File: code/modules/ruins/spaceruin_code/bigderelict1.dm (hunk 2 above)
Comment (Capsandi) [reply code/modules/ruins/spaceruin_code/bigderelict1.dm:11]: Thanks for catching this

File: code/game/area/areas/ruins/space.dm (hunk 1 above)
Comment (Capsandi) [reply code/game/area/areas/ruins/space.dm]: Didn't mean to, thanks for the catch

**Comment by Capsandi** [comment]
> its way more boring imo. instead of creatures you cant find anywhere else you fight spiders that you can already find in like 2 other ruins, theres no more abandoned crates, removing a puzzle that gave some nice random loot if you were good, the cool idea of a broken shuttle that docked there got removed
> using new mapping practices for ruins is nice, but sacrificing how interesting the ruin is for that is lame

Creatures are just simple mobs. Scrawny Spiders are aswell but non-scrawny ones(such as the new Mr. Tiggles) are venomous. Also the only other ruin I could think of with spiders on it would be the delta whiteship, unless im forgetting something.
Abandoned crates still spawn, they are represented by the lootbox icons, sorry for the confusion
As for the docked shuttle, I couldn't find a good layout which integrated the idea without introducing a large empty room similar to those I was trying to get rid of

> QoL is for making things more convenient and easier to use not changing them at a substantial level.

Yea I don't know what I was thinking with that, I suppose ill slap the feature label on there instead?

**Comment by notfrying1pans** [comment]
i don't see why medical crates were removed, everything else is fine though.

**Comment by tralezab** [comment]
It's worse

**Comment by Yenwodyah** [comment]
This looks way easier than the old one. 4 of the spiders you can laser through a window, or just break the window and let them die to pressure damage. Flavor-wise, you turned it from a unique location with a clear story into... basically kilo maint in space? How is this supposed to be an improvement?

**Comment by bluetshirtguy** [comment]
It's just a dull tiny box now, I get wanting to tone down the gear found but jesus man there's nothing to even explore in this ruin you made.

**Comment by CRITAWAKETS** [comment]
this isnt even close in any way, shape or form aside from just being a space ruin

**Comment by Valorium-RB** [comment]
I hate to pile on, but I feel like most of the other ruins seem to, well, BE something specific. An armory, a research facility, a vault, a diner, and so on.  This ruin doesn't really seem to BE anything specific at a glance.  It's a handful of random maintenance rooms, a security post guarding what seems to be a room with a table and vending machines and a room with some random crates and freezers in the middle.  Now this ruin has, well, literally no noteworthy loot. It has maint loot, an EVA suit that you already have, an M1911 with no mag (so basically a paperweight, go figure) and some blood bags.  I cannot imagine a ruin wherein the best, most interesting loot is an _electrical toolbox_, but here we are. Solid no from me, sorry.

**Comment by Capsandi** [comment]
Alright I suppose I will revisit this in the future, though perhaps the idea of a outpost entirely focused on trading conflicts at a conceptual level with my efforts to turn it into something other than piles of free gear in space.

**Comment by Valorium-RB** [comment]
If you wanted to do something like that, my recommendation would probably be either a) something similar to the Goonstation-style Listening Post where you can buy a few select Syndicate items via vendor (or, really, just a room with a bunch of rare, custom vending machines selling random loot, which would give use to the bundles of cash found all over space) or b) a ruin with a new ghostrole (which I doubt would go over well considering spessbar got declined ages ago) based around selling wares in a room that can't be broken into, a la Baystation's Trader role.  Both would be good examples to look at for guidance if you want a trade-based ruin that isn't also just a free loot pile.

**Comment by bluetshirtguy** [comment]
Oh, before I forget have you thought about working with TG's lore man eob with whatever future redesign you have in mind? I think it's a good chance to add a ruin with lore and guff.

**Comment by Capsandi** [comment]
Nah, I was just trying to take the secgear out of the ruin while improving the layout, which I cant say I didn't do. I closed it because I forgot spiders took pressure damage. Now that I think of it, a full removal of the ruin would be a better route to go through at some point
